src/components/BookForm.js

```diff
@@ -1,15 +1,30 @@
 import '../App.css';
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
 
-const BookForm = () => (
-  <div>
-    <form action="/action_page.php">
-      <input type="text" name="title" value="Add your title" />
-      <br />
-      <input type="text" name="Author" value="Author's name" />
-      <input type="submit" value="Submit" />
-    </form>
-  </div>
-);
+const BookForm = () => {
+  const dispatch = useDispatch();
+
+  const handleSubmit = (e) => {
+    const bookTitle = e.target.title.value;
+    const bookAuthor = e.target.author.value;
+    const id = Date.now();
+    e.target.title.value = '';
+    e.target.author.value = '';
+    dispatch(addBook({ title: bookTitle, author: bookAuthor, id }));
+  };
+
+  return (
+    <div>
+      <form onSubmit={(e) => { e.preventDefault(); handleSubmit(e); }}>
+        <input type="text" name="title" placeholder="Add your title" />
+        <br />
+        <input type="text" name="author" placeholder="Author's name" />
```

Comment:
- [ ] Currently, your app does not add new books or remove any book from the current list. Ensure to make this feature fully functional.
- [ ] These inputs should have their local state update every time the input field changes the value. Use the `onChange` attribute. This is one of the reasons I am not able to add new books.